src/Common/src/CoreLib/System/IO/Win32Marshal.cs

```diff
@@ -72,21 +72,6 @@ internal static int MakeHRFromErrorCode(int errorCode)
             return unchecked(((int)0x80070000) | errorCode);
         }
 
-        /// <summary>
-        /// Returns a Win32 error code for the specified HRESULT if it came from FACILITY_WIN32
-        /// If not, returns the HRESULT unchanged
-        /// </summary>
-        internal static int TryMakeWin32ErrorCodeFromHR(int hr)
-        {
-            if ((0xFFFF0000 & hr) == 0x80070000)
-            {
-                // Win32 error, Win32Marshal.GetExceptionForWin32Error expects the Win32 format
-                hr &= 0x0000FFFF;
-            }
-
-            return hr;
-        }
```

Comment:
This is not dead code. It's used by System.Runtime.WindowsRuntime.

Reply:
I did a full build and tests and had no issues. Is there something else I should be doing to determine if it's OK to remove?

Reply:
Did you do a `build --allconfigurations`?

Reply:
Excellent. Thanks.